src/Workspaces/Remote/ServiceHub/Services/DiagnosticAnalyzer/DiagnosticComputer.cs

```diff
@@ -57,8 +57,7 @@ internal class DiagnosticComputer
             _analysisKind = analysisKind;
             _analyzerInfoCache = analyzerInfoCache;
 
-            // We only track performance from primary branch. All forked branch we don't care such as preview.
-            _performanceTracker = project.IsFromPrimaryBranch() ? project.Solution.Services.GetService<IPerformanceTrackerService>() : null;
+            _performanceTracker = project.Solution.Services.GetRequiredService<IPerformanceTrackerService>();
```

Comment:
i don't see the value that in this.  Why would it not matter what the performance was getting diagnostics in a preview workspace?